src/util/simple_timer.h

```diff
@@ -64,7 +64,22 @@
     // Finally, the ANSI C11 clock() function is used if no other timer
     // is available.
 
-#endif
+#endif // if _OPENMP
+
+typedef double simple_timer_t[2];
```

Comment:
so now `simple_timer_t` is an alias to double[2], I think it would be nice to update both `simple_tic` and `simple_toc` to use it. e.g.
```c
void simple_tic         // returns current time in seconds and nanoseconds
(
    simple_timer_t tic      // tic [0]: seconds, tic [1]: nanoseconds
) ;
double simple_toc           // returns time since last simple_tic
(
     const simple_timer_t tic    // tic from last call to simple_tic
 ) ;
```